spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/stream/AppDeployerStreamDeployer.java

```diff
@@ -182,18 +181,18 @@ public DeploymentState calculateStreamState(String streamName) {
 	}
 
 	@Override
-	public Map<StreamDefinition, DeploymentState> state(List<StreamDefinition> streamDefinitions) {
+	public Map<StreamDefinition, DeploymentState> streamsStates(List<StreamDefinition> streamDefinitions) {
 		Map<StreamDefinition, List<String>> deploymentIdsPerStream = streamDefinitions.stream()
 				.collect(Collectors.toMap(Function.identity(),
 						sd -> sd.getAppDefinitions().stream().map(
 								sad -> deploymentIdRepository.findOne(DeploymentKey.forStreamAppDefinition(sad)))
 								.collect(Collectors.toList())));
 
-		// Map from app deployment id to state
+		// Map from app deployment id to streamsStates
```

Comment:
to `DeploymentState` would be better - though this change is a result of a search/replace operation....as well as a few other spots.